contracts/cw-proposal-multiple/src/voting_strategy.rs

```diff
@@ -1,24 +1,30 @@
+use cosmwasm_std::Uint128;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
-use voting::threshold::{validate_quorum, PercentageThreshold, ThresholdError};
+use voting::threshold::{validate_count, validate_quorum, PercentageThreshold, ThresholdError};
+
+/// Determines the way votes are counted.
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum MultipleProposalThreshold {
```

Comment:
I agree it may be a good idea to have a separate Threshold type for multiple choice to avoid confusion because of the differences in the ways thresholds are used between the two proposal modules. however, can we move this to `packages/cwd-voting/src/threshold.rs`? Also, can we change the names to `PercentageQuorum` and `AbsoluteCountQuorum` just to make it super clear how they are functioning (all MC thresholds will only function as a quorum after which the highest vote will be selected)